bluemira/base/reactor_config.py

```diff
@@ -35,6 +35,25 @@ class ReactorConfig:
     config_path: str | dict
         The path to the config JSON file or a dict of the data.
 
+    global_params_type: Type
```

Comment:
sphinx auto copies the types into the docstring of the function so we only need to write the types in one place on the function argument definitions (less likely to get out of sync). Can we remove the types from the docstring here and add them all onto the init?

Reply:
Ah that's great, was always annoying to copy them into the docstring